coderd/prometheusmetrics/prometheusmetrics_test.go

```diff
@@ -176,13 +185,24 @@ func TestUsers(t *testing.T) {
 				}
 
 				for _, metric := range metrics[0].Metric {
-					if tc.Count[*metric.Label[0].Value] != int(metric.Gauge.GetValue()) {
+					if tc.Count[database.UserStatus(*metric.Label[0].Value)] != int(metric.Gauge.GetValue()) {
 						return false
 					}
 				}
 
 				return true
-			}, testutil.WaitShort, testutil.IntervalSlow)
+			}
+
+			require.Eventually(t, checkFn, testutil.WaitShort, testutil.IntervalFast)
```

Comment:
Hhmm, do we need `require.Eventually` with quartz now implemented (thank you for doing this!)?

Reply:
It was my assumption that `AdvanceNext().MustWait()` only waits for the ticker to fire, then the actual gathering code runs async. I guess I could technically make it block until the end using `AfterFunc`, but that felt kinda janky.